src/Entities/Order.php

```diff
@@ -56,59 +46,39 @@ public function __construct(
         if ($additionalProperties) $this->filterAdditionalProperties($additionalProperties);
     }
 
-    public function getId(): string|null
+    public function getId(): BaseField|null
     {
-        return isset($this->id) ? $this->id->get() : null;
+        return $this->id;
     }
 
-    public function getStatus(): string|null
+    public function getStatus(): Status|null
     {
-        return $this->status?->get();
+        return $this->status;
     }
 
-    public function getClient(): array
+    public function getClient(): Client
     {
-        return $this->client?->toArray();
+        return $this->client;
     }
 
-    public function getCustomer(): array
+    public function getCustomer(): Customer
     {
-        return $this->customer->toArray();
+        return $this->customer;
     }
 
-    public function getAmount(): int
+    public function getAmount(): Amount
     {
-        return $this->amount->get();
+        return $this->amount;
     }
 
     public function getOrderLines(): ?OrderLines
     {
         return $this->orderLines;
     }
 
-    public function getReturnUrl(): string
+    public function getExtra(): Extra
     {
-        return $this->returnUrl->get();
-    }
-
-    public function getWebhookUrl(): string
-    {
-        return $this->webhookUrl->get();
-    }
-
-    public function getMerchantOrderId(): string
-    {
-        return $this->merchantOrderId->get();
-    }
-
-    public function getExtra(): array
-    {
-        return $this->extra?->toArray();
-    }
-
```

Comment:
Are functions like getMerchantOrderId or getWebhookUrl moved somewhere? Or they were just deleted?

Reply:
Removed, because i don't see cases for usages. Do you see where it could be useful?

Reply:
MerchantOrderId could be used in some messages to customer. On success page or in emails.
Getting webhook url from order, probably, useless, I agree

Reply:
Ye, you might be right. I think I just missed up `merchant_order_id` property at all at this entity. Will implement in next commit.